src/tests/performance/performance.py

```diff
@@ -0,0 +1,59 @@
+import subprocess
+import json
+import sys
+import os
+
+from pathlib import Path
+
+class StudyList(object):
+
+    def __init__(self, name, path):
+        self.name = name
+        self.path = Path(path).resolve()
+        self.outFile = name + "-metrics.txt"
+
+    def run_metrics(self, exe):
+        res = subprocess.run(["/bin/time" , "-o", self.outFile, "-v", exe, self.path])
+        assert (res.returncode == 0), "The exec failed for study: " + str(self.path)
+
+    def get_memory(self):
+        ps = subprocess.run(["sed -n -e 's/^.*Maximum.*: //p' " + self.outFile], shell=True, capture_output=True)
+        return int(ps.stdout)
+
+    def get_time(self):
+        ps = subprocess.run(["sed -n -e 's/^.*User time.*: //p' " + self.outFile], shell=True, capture_output=True)
+        ps2 = subprocess.run(["sed -n -e 's/^.*System time.*: //p' " + self.outFile], shell=True, capture_output=True)
+        return float(ps.stdout) + float(ps2.stdout)
```

Comment:
You could run only once. You just need the "User time". Also, do the parsing in Python, don't use sed.